cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/vendor/github.com/coreos/etcd/cmd/main.go

```diff
@@ -1 +0,0 @@
-../main.go
```

Comment:
> the changes to this package surprised me

weird, but it looks like it will hold (symlink to file with same content) and be fixable immediately after.